hdfs-commons/src/main/java/org/apache/mesos/hdfs/util/FailureUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * Failure utilities.
+ */
+public class FailureUtils {
+  private static final Log log = LogFactory.getLog(FailureUtils.class);
+
```

Comment:
Ideally, let's be consistent with executors and the scheduler, one way or another.

 I do prefer changing the gradle settings and being very careful when we decide to exit. If we break this rule often and have a reason behind it, I'm not sure we should be suppressing warnings everywhere. We could still carefully comment the code. That said, I see your point, maybe we should always suppress the warnings and document it, but let's be consistent.